gdx/src/com/badlogic/gdx/maps/tiled/AtlasTmxMapLoader.java

```diff
@@ -252,20 +255,49 @@ protected TiledMap loadMap (Element root, FileHandle tmxFile, AtlasResolver reso
 
 		for (int i = 0, j = root.getChildCount(); i < j; i++) {
 			Element element = root.getChild(i);
-			String elementName = element.getName();
-			if (elementName.equals("properties")) {
-				loadProperties(map.getProperties(), element);
-			} else if (elementName.equals("tileset")) {
-				loadTileset(map, element, tmxFile, resolver);
-			} else if (elementName.equals("layer")) {
-				loadTileLayer(map, map.getLayers(), element);
-			} else if (elementName.equals("objectgroup")) {
-				loadObjectGroup(map, map.getLayers(), element);
-			}
+			loadLayer(map, map.getLayers(), element, tmxFile, resolver);
 		}
 		return map;
 	}
 
+    protected void loadLayer(TiledMap map, MapLayers parentLayers, Element element, FileHandle tmxFile, AtlasResolver resolver) {
+        String elementName = element.getName();
+        if (elementName.equals("group")) {
+            loadTileGroup(map, parentLayers, element, tmxFile, resolver);
+        } else if (elementName.equals("properties")) {
```

Comment:
A bit of a nitpick but "properties" isn't a layer, so having it be read in the `loadLayer` method seems incorrect.

Reply:
You're right. The reason why I didn't change it is to kep it consistent with BaseTmxMapLoader.loadLayer(...) where nearly the same stuff happens.

Reply:
`BaseTmxMapLayer#loadLayer` doesn't handle properties.

Reply:
So, what would you suggest? loadLayerAndProperties?

Reply:
I'll just spend some time refactoring this weekend to get into a more sane place as far as duplication goes.

Reply:
That's great! Thanks a lot!